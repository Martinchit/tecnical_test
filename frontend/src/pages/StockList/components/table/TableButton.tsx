import styled from 'styled-components';
import { Button } from 'react-bootstrap';

export const TableButton = styled(Button)`
  margin-right: 3%;
  width: 40%;

  @media (max-width: 767px) {
    width: 100%;
    margin-top: 5%;
    font-size: 0.8rem;
  }

  &:focus {
    outline: none;
    box-shadow: none;
  }
`;
