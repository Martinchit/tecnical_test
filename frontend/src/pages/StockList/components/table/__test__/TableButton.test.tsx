import React from 'react';
import { shallow } from 'enzyme';

import { TableButton } from '../TableButton';

describe('TableButton', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<TableButton />);
    expect(wrapper).toMatchSnapshot();
  });
});
