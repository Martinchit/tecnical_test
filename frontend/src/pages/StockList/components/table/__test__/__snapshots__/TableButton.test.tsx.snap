// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableButton renders correctly 1`] = `
<Button
  active={false}
  className="sc-AxjAm XCNff"
  disabled={false}
  type="button"
  variant="primary"
/>
`;
